Bottom: 22fee6c95765dc6576bb4dce15b9e2e5d4a34dcb
Top:    325c317b539d82fb72cde882c9b5521944a08365
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-29 14:15:22 -0500

Refresh of all-docker_volume

---
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 193767b9..aa81c6ec 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -132,8 +132,6 @@
     name: '{{ mantisbt_db_container_name }}'
     detach: true
     restart_policy: '{{ docker_restart_policy }}'
-    volumes_from:
-      - '{{ mantisbt_db_dv_name }}'
     image: '{{ mantisbt_db_image_name }}'
     volumes:
       - '{{ mantisbt_db_cv_name }}:/etc/mysql/conf.d:z'
